### PR TITLE
Run directory updates related to GEOS-IT

### PR DESCRIPTION
**Pull request by lizziel**
### Name and Institution (Required)

Name: Lizzie Lundgren
Institution: Harvard University

### Describe the update

This PR contains several updates related to GEOS-IT meteorology and GCHP.
- Added GEOS-IT processed lat-lon fields as a valid option when creating GCHP run directories. Previously these files were not available at WashU. Now there are at least a few days to test with.
- Changed GCHP sample run scripts to not print script execution commands to log. This reduces the amount of output sent to log. The option is still present in the run scripts but is commented out. It can be uncommented to debug problems with the run script.
- Changed offline emissions grid resolution templates in config files to be more descriptive. This applies to both GC-Classic and GCHP. It is a no-diff update but helps with clarify when reading the met files used to fill in `HEMCO_Config.rc` and `ExtData.rc` when creating a run directory. `RUNDIR_MET_NATIVE_RES` is now `RUNDIR_OFFLINE_EMIS_RES` and `RUNDIR_MET_LAT_RES` is now `RUNDIR_OFFLINE_EMIS_LAT_RES`. Entry `RUNDIR_MET_LON_RES` is deleted since it is not used.
- Fixed GEOS-IT native lat-lon filenames used for clusters other than discover. This now enables GCHP run directories using GEOS-IT native lat-lon meteorology to work.
- Fixed offline emission paths set when using GEOS-IT meteorology. This now enables GCHP standard simulations using GEOS-IT meteorology to run. Previously they failed due to incorrect population of the offline emission paths in `ExtData.rc`.

### Expected changes

This is a zero diff update

### Related Github Issue(s)

closes https://github.com/geoschem/GCHP/issues/379

### CODE REVIEW

File: run/shared/settings/geosit/geosit.native_ll.txt [hunk 1]
```diff
@@ -29,41 +29,41 @@ ${RUNDIR_MET_EXTDATA_PRIMARY_EXPORTS_FOR_ADVECTION}
...
-TO3       Dobsons    N Y F0;003000 none  none TO3      ./MetDir/%y4/%m2/%d2/GEOS.it.asm.slv_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 1995-01-01T00:30:00P01:00
-TROPP     Pa         N Y F0;003000 none  0.01 TROPPT   ./MetDir/%y4/%m2/%d2/GEOS.it.asm.slv_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 1995-01-01T00:30:00P01:00
-TSKIN     K          N Y F0;003000 none  none TS       ./MetDir/%y4/%m2/%d2/GEOS.it.asm.slv_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 1995-01-01T00:30:00P01:00
-U10M      m_s-1      N Y F0;003000 none  none U10M     ./MetDir/%y4/%m2/%d2/GEOS.it.asm.slv_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 1995-01-01T00:30:00P01:00
-V10M      m_s-1      N Y F0;003000 none  none V10M     ./MetDir/%y4/%m2/%d2/GEOS.it.asm.slv_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2Z.nc4 1995-01-01T00:30:00P01:00
+EFLUX     W_m-2      N Y F0;003000 none  none EFLUX    ./MetDir/%y4/%m2/%d2/GEOS.it.asm.flx_tavg_1hr_glo_L576x361_slv.GEOS5294.%y4-%m2-%d2T%h2%n2.V01.nc4 1995-01-01T00:30:00P01:00
```
Comment (yantosca) [inline run/shared/settings/geosit/geosit.native_ll.txt:32]: Q: Is the 1995-01-01T00:30:00P01:00 the reference time?

**Review by yantosca — CHANGES_REQUESTED** [review]
@lizziel: Just needs a changelog entry and then it's good to merge.  Thanks for this!

**Review by yantosca — APPROVED** [review]
@lizziel: I pushed the changelog update.  Now moving to integration tests

**Comment by lizziel** [comment]
Oops, forgot the changelog again! I'm going to push an update on top of yours for the changelog.

**Comment by lizziel** [comment]
Looks like I already did a changelog but just forgot to push. I will fix the branch.

**Comment by lizziel** [comment]
All set now. I had rebased on dev/no-diff-to-benchmark but forgot to push that too. This PR now has my changelog updates and no conflicts with dev/no-diff-to-benchmark.

**Review by yantosca — APPROVED** [review]
I've pushed a change to make sure that `RUNDIR_OFFLINE_EMIS_RES` and `RUNDIR_OFFLINE_EMIS_LAT_RES` are defined for MERRA-2 meteorology.  Also the GCHP and GCClassic `HEMCO_Config.rc.fullchem` template files needed to be updated to replace `RUNDIR_MET_NATIVE_RES` with `RUNDIR_OFFLINE_EMIS_RES` and `RUNDIR_MET_LAT_RES` with `RUNDIR_OFFLINE_EMIS_LAT_RES`.

**Comment by yantosca** [comment]
All GCClassic and GCHP integration tests passed. 

All GCHP integration tests were zero-diff w/r/t 14.3.0.  All GCClassic integration tests were zero-diff w/r/t 14.3.0, except:
- APM (known parallelization issue)
- RRTMG (numerical noise in RRTMG diagnostic collection output)

**Pull request merged**